docker/bookkeeper/entrypoint.sh

```diff
@@ -20,11 +32,14 @@ BK_CLUSTER_NAME=${BK_CLUSTER_NAME:-"bookkeeper"}
 BK_LEDGERS_PATH="/${PRAVEGA_PATH}/${PRAVEGA_CLUSTER_NAME}/${BK_CLUSTER_NAME}/ledgers"
 BK_DIR="/bk"
 
+create_dir "${BK_journalDirectories}"
+create_dir "${BK_ledgerDirectories}"
+
 export BOOKIE_PORT=${BOOKIE_PORT}
 export BK_zkServers=${BK_zkServers}
 export BK_metadataServiceUri=zk://${ZK_URL}${BK_LEDGERS_PATH}
-export BK_journalDirectory=${BK_DIR}/journal
-export BK_ledgerDirectories=${BK_DIR}/ledgers
+export BK_journalDirectories=${BK_journalDirectories:-${BK_DIR}/journal}
```

Comment:
We have found the issue with this change in which, if `journalDirectories` is not specified in the Pravega manifest, the deprecated `journalDirectory` (and `ledgerDirectory`) parameter is taken as the path to store Bookkeeper data. Note that default value for these deprecated parameters is currently `/data/xxx`, instead of `/bk/xxx`. The consequence of this is that the Bookie can end up storing data in the local storage of the pod, instead of using the volumes defined for that purpose. We may need to override the default values for `journalDirectory` and `ledgerDirectory` to avoid this problem if someone forgets to specify `journalDirectories`/`ledgerDirectories`.

Reply:
Thank Raul, that's a good point. However, I think here it will provide the default value `${BK_DIR}/journal`,which is `/bk/journal`, to the Bookkeeper if `BK_journalDirectories` is not specified. The reason that we are getting `/data/bk` is because `/data/bk` is the default value in official Bookkeeper scirpt and I didn't provide our default value in our experiment image(sorry about that!).